v2/http-middleware/security.go

```diff
@@ -224,24 +230,32 @@ func AuthorizeMiddleware(authorizer Authorizer) mux.MiddlewareFunc {
 					WithField("url", req.URL.String()).
 					Error("Couldn't extract User ID from context")
 
+				responseBody := GetInternalServerErrorResponseBody(http_model.ErrResponseInternalServerError, secConfig)
+
 				http_server.WriteJSONResponse(
-					ctx, w, req, http.StatusInternalServerError, http_model.ErrResponseInternalServerError,
+					ctx, w, req, http.StatusInternalServerError, responseBody,
 				)
 				return
 			}
 
 			isAuthorized, err := checkAuthorization(ctx, req, authorizer, userID, secConfig.authorizations)
 			var httpErr *http_model.HTTPError
 			if errors.As(err, &httpErr) {
-				http_server.WriteJSONResponse(ctx, w, req, httpErr.StatusCode, httpErr.Message())
+				if secConfig.responses != nil {
+					http_server.WriteJSONResponse(ctx, w, req, http.StatusInternalServerError, secConfig.responses.InternalErrorResponse())
+				} else {
```

Comment:
this else is not needed?

Reply:
You mean by adding a return in the if body?

Reply:
Yes, I just assumed that it was already there ^^